extpsdk/build.gradle

```diff
@@ -11,7 +11,7 @@ ext {
         pypi_repo = 'testpypi'
     }
     if (!project.rootProject.hasProperty('vantiqConnectorSdkVersion')) {
-        vantiqConnectorSdkVersion = '1.1.13'
```

Comment:
moving up an increment since changing Python version.  No real API change, and things will probably still work with older versions.  But to be safe...